tutorials/intermediate/artists.py

```diff
@@ -433,8 +451,8 @@ class in the matplotlib API, and the one you will be working with most
 #     None
 #
 #     # and the transformation instance is set to the "identity transform"
-#     In [265]: print(rect.get_transform())
-#     <Affine object at 0x13695544>
+#     In [265]: print(rect.get_data_transform())
```

Comment:
Note, `get_transform` is wrong, because it shows the re-scaling of a unit rectangle to the expected size, and not the identity transform that the tutorial is talking about here.